File: Python/Fib.py
def fibonacci(n):
    if n <= 0:
        return []
    elif n == 1:
        return [0]
    elif n == 2:
        return [0, 1]
    else:
        fib_series = fibonacci(n - 1)
        fib_series.append(fib_series[-1] + fib_series[-2])
        return fib_series

# Get user input for the number of terms
num_terms = int(input("Enter the number of Fibonacci terms to generate: "))

# Print the Fibonacci series
result = fibonacci(num_terms)
print("Fibonacci series up to",num_terms,"terms:-",result)

File: C/Fib.c
#include <stdio.h>

void fibonacci(int n){
    int fib[n + 2];
    fib[0] = 0;
    fib[1] = 1;

    for (int i = 2; i <= n; i++){
        fib[i] = fib[i - 1] + fib[i - 2];
    }

    printf("Fibonacci Series:-- ");

    for (int i = 0; i < n; i++){
        printf("%d, ", fib[i]);
    }
}

int main(){
    int n;

    printf("Enter the no. upto where you want to print Fibonacci Series:- ");
    scanf("%d", &n);

    fibonacci(n);

    return 0;
}